### DISCUSSION TITLE
how to fetch the CMS data via API instead of passing it into Laravel blade?

### DISCUSSION THREAD

**Discussion by lamualfa** [LavaLite/cms#375] [Q&A]
I want to integrate my Svelte Kit project with Lavalite as its CMS. Exporting Svelte Kit project to Laravel Blade is difficult. Instead, I use API to get data from CMS and use it within Svelte Kit project.

The question is, there's any documentation about the use of API in Lavalite? I can't find anything about that in Google.

**Comment by sukruatmcc** [comment]
@lamualfa Laravel API https://laravel.com/api/9.x/ buradan inceleyebilirsiniz.
